src/object_sync/hello.cpp

```diff
@@ -74,15 +71,20 @@ namespace object_sync {
   // NAN_METHOD is applicable to methods you want to expose to JS world
   NAN_METHOD(HelloObject::hello) {
 
-    // Note: a HandleScope is automatically included inside NAN_METHOD (See the docs at NAN that say:
+    // Note: a HandleScope is automatically included inside NAN_METHOD. See the docs at NAN that say:
     // 'Note that an implicit HandleScope is created for you on JavaScript-accessible methods so you do not need to insert one yourself.'
```

Comment:
@springmeyer Is this comment still relevant? Or does it belong below in Init where the Handlescope is created?